packages/core/src/components/responsive-element/responsive-element.tsx

```diff
@@ -1,8 +1,10 @@
-import { Component, ComponentInterface, Element, Host, State, h } from '@stencil/core';
+import { Component, ComponentInterface, Element, Event, EventEmitter, h, Host, Method, State } from '@stencil/core';
+
+export type ResponsiveElementSize = 'small' | 'medium' | 'large';
```

Comment:
Ik denk dat we dit type en `ElementSize` beter onder kunnen brengen in een nieuwe file `responsive-element.inferfaces.ts`

Reply:
Done